src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -721,6 +721,182 @@ RppStatus rppt_resize_crop_mirror_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
+                             RpptDescPtr srcDescPtr,
+                             RppPtr_t dstPtr,
+                             RpptDescPtr dstDescPtr,
+                             RpptROIPtr roiTensorPtrSrc,
+                             RpptRoiType roiType,
+                             rppHandle_t rppHandle)
+{
+    RppLayoutParams srcLayoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+
+    if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::U8))
+    {
+        RpptImagePatchPtr internalDstImgSizes = (RpptImagePatch *) calloc(dstDescPtr->n, sizeof(RpptImagePatch));
+        RpptROI *internalRoiTensorPtrSrc = (RpptROI *) calloc(dstDescPtr->n, sizeof(RpptROI));
+        for(int i=0;i<dstDescPtr->n;i++)
+        {
+            internalDstImgSizes[i].width = internalRoiTensorPtrSrc[i].xywhROI.roiWidth = roiTensorPtrSrc[i].xywhROI.roiWidth / 8;
+            internalDstImgSizes[i].height = internalRoiTensorPtrSrc[i].xywhROI.roiHeight = roiTensorPtrSrc[i].xywhROI.roiHeight / 8;
+        }
+        RpptDescPtr interDescPtr = dstDescPtr;
+        unsigned long long interBufferSize = (unsigned long long)interDescPtr->h * (unsigned long long)interDescPtr->w * (unsigned long long)dstDescPtr->c * (unsigned long long)dstDescPtr->n;
```

Comment:
Since the size of intermediate images will be width / 8 and height / 8
strides also needs to be updated as
interDescPtr->h \= 8;
interDescPtr->w \= 8;
interDescPtr->nStride = interDescPtr->h * interDescPtr->w * interDescPtr->c;

Reply:
Sure will add these, Outputs were fine even without these modifications for HOST so left them as such.

Reply:
Outputs are getting corrupted if i add these for HOST, so removed them.